apps/sensenet/src/components/dialogs/permission-editor-dialog.tsx

```diff
@@ -0,0 +1,396 @@
+              }}
+            />
+          </ListItem>
+          <Divider />
+          <ListItem>
+            <ListItemText primary={localization.permissionEditor.localOnly} className={classes.disabled} />
+            <Switcher checked={false} size="small" disabled onClick={() => {}} />
+          </ListItem>
+        </List>
+        <div className={clsx(classes.column, classes.rightColumn)}>
+          <div className={classes.permissionContainer}>
+            <List>
+              {getPermissionsFromGroupName(actualGroup).map((selectedGroupPermission: keyof PermissionRequestBody) => {
+                return (
+                  <ListItem key={selectedGroupPermission} onClick={() => {}}>
+                    <ListItemText
+                      disableTypography
+                      primary={selectedGroupPermission}
+                      className={clsx({
+                        [classes.disabled]: isPermissionDisabled(selectedGroupPermission),
+                      })}
+                    />
+                    <Switcher
+                      checked={
+                        (responseBody[selectedGroupPermission] !== undefined &&
+                          responseBody[selectedGroupPermission] === PermissionValues.allow) ||
+                        (responseBody[selectedGroupPermission] === undefined &&
+                          props.entry.permissions[selectedGroupPermission]?.value === 'allow')
+                      }
+                      disabled={isPermissionDisabled(selectedGroupPermission)}
+                      size="small"
+                      onClick={() => {
+                        const localResponseBody = { ...responseBody }
+                        if (responseBody[selectedGroupPermission] === PermissionValues.allow) {
+                          Object.assign(localResponseBody, {
+                            [selectedGroupPermission]: PermissionValues.undefined,
+                          })
+                        } else {
+                          Object.assign(localResponseBody, {
+                            [selectedGroupPermission]: PermissionValues.allow,
+                          })
+                        }
+                        setResponseBody(localResponseBody)
+                      }}
+                    />
+                  </ListItem>
+                )
+              })}
+            </List>
+          </div>
+          <DialogActions className={classes.dialogActions}>
+            <Button
+              aria-label={localization.forms.cancel}
+              className={globalClasses.cancelButton}
+              onClick={closeLastDialog}>
+              {localization.forms.cancel}
+            </Button>
+            <Button
+              aria-label={localization.forms.submit}
+              color="primary"
+              variant="contained"
+              autoFocus={true}
+              onClick={async () => {
+                if (props.path) {
```

Comment:
should we give some feedback if no path given? or is that a valid scenario?

Reply:
You cannot open the permission view if there is no currentContent (because of context-menu - it has to reqest the actions so it also need a content), so it should not be undefined